src/canvas/SelectableCanvas.ts

```diff
@@ -812,8 +791,13 @@ export class SelectableCanvas<EventSpec extends CanvasEvents = CanvasEvents>
     // until we call this function specifically to search inside the activeGroup
     while (i--) {
       const objToCheck = objects[i];
+      // this strange logic indicates a severe bug in containsPoint
       const pointerToUse = objToCheck.group
-        ? this._normalizePointer(objToCheck.group, pointer)
+        ? sendPointToPlane(
+            pointer,
+            invertTransform(this.viewportTransform),
+            objToCheck.group.calcTransformMatrix()
+          )
         : pointer;
```

Comment:
How the bug manifest? The fact that the code is not your liking doesn't mean is bugged.

Reply:
It does.
The method accepts points in different cases, under group with viewportTransform removed from the point and if not under group then the point in the html plane.
If that is not a bug in design I do not now what a bug is.
I do not plan to touch this because #8767 fixed that and I am tired of getting back to thing I fixed

Reply:
It is a bug because I can't use containsPoint, no one can without copying this code because it makes no sense